### PR TITLE
GRANT: deploy to prod config

### PR DESCRIPTION
**Pull request by alva-fresh**
# Summary:
- added config for prod deploy;
- added a new workflow to trigger deployments on `pull_request` to _main_;

### CODE REVIEW

File: .github/workflows/bcat-prod-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      name: prod
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./.pipeline
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
```
Comment (coryatwater-FW) [inline .github/workflows/bcat-prod-deploy.yml:38]: `16`?

**Review by coryatwater-FW — APPROVED** [review]
one comment, but looks good to me!

File: .pipeline/lib/config.js [hunk 2]
```diff
@@ -59,6 +59,23 @@ const phases = {
...
     version: `${version}`,
   },
+  prod: {
+    api_cpu: '250m',
+    api_memory: '512Mi',
+    changeId: changeId,
+    client_cpu: '150m',
+    client_memory: '512Mi',
+    dbName: 'bcat',
+    host: `bcat-bfe2da-prod.apps.silver.devops.gov.bc.ca`,
```
Comment (coryatwater-FW) [inline .pipeline/lib/config.js:69]: do we need to change this to match Yuri's link?

**Comment by alva-fresh** [comment]
We updated deployment workflow in #83; closing this

**Pull request closed without merging**